### DISCUSSION TITLE
Add additional checkbox to checkout with Gutenberg editor

### DISCUSSION THREAD

**Discussion by dgdb1** [woocommerce/woocommerce#51978] [Checkout Flow]
In some countries where WooCommerce is used, the order form should include an additional checkbox that confirms the privacy policy. It cannot be combined with the terms and conditions. There must be two separate consents to provide.

Unfortunately, in the new editor, there is no option to add an additional consent checkbox. Moreover, I cannot add this through the function.php file.

How can I add an additional field using the checkout page editor?

**Comment by alexflorisca** [comment]
You can add additional fields to the checkout using the [additional fields API](https://developer.woocommerce.com/docs/cart-and-checkout-additional-checkout-fields/). There is currently no way to add an extra checkbox using the Gutenberg editor only. You can require the checkbox for the terms and conditions by clicking on it and selecting the option from the inspector controls

<img width="1177" alt="Screenshot 2024-11-11 at 16 25 54" src="https://github.com/user-attachments/assets/efe4771c-d1c5-40ea-93f2-32719de153a5">
